testprojects/src/python/interpreter_selection/python_3_selection_testing/lib_py23.py

```diff
@@ -0,0 +1,21 @@
+# coding=utf-8
```

Comment:
Remind me why you created new files for this? I would prefer if you just reworked the existing files to fit your testing requirements rather than creating these new files. Then you could just add more targets.

This route would require you to update the test scripts here (`test_py{2, 3}`) to ensure they are passing and checking for the proper return values ("Python3" instead of ascii, or whatever).

Reply:
Because the original use case was working with libraries that are Python2/3 compatible, and the targets and files already there are specific to each version.
Arguably, we could convert either the 2 or 3 version to be 2/3 compatible, but I didn't want to modify any existing tests and behaviors.

Why would the tests need to be changed? They still use the same files and targets, the reason I created new files was to avoid that. Maybe there's something I'm missing?

I rather have Python2, Python3, and Python2/3 files and targets, so we can be sure we're testing against  every use case.

Reply:
Sounds good to me! I was conflating these files with Twitter-internal files that support this testing.